<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="4" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayoutPlainText">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>6</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>6</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="5">
       <layout class="QVBoxLayout" name="verticalLayoutSearch">
        <item>
         <widget class="QSpinBox" name="spinBoxSearchSizeFont">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSearchBold">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Pogrubiona</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSearchItalic">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Kursywa</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" colspan="7">
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="labelSearchFont">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Czcionka
wyszukiwania</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="pushButtonSettings">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>Pokaż
ustawienia</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonRefresh">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Odświeża listę odczytaną w archiwum&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Odśwież</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRead">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Odczytuje listę wszystkich plików na dysku Archiwum&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Odczytaj listę
plików</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelListFont">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Czcionka
listy</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <layout class="QVBoxLayout" name="verticalLayoutList">
        <item>
         <widget class="QSpinBox" name="spinBoxListSizeFont">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxListBold">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Pogrubiona</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxListItalic">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Kursywa</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBoxOpenDirectory">
          <property name="text">
           <string>Otwórz katalog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonTotalCmd">
          <property name="text">
           <string>TotalCMD</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupAppType</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonLeftSide">
            <property name="text">
             <string>Okno lewe</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupSide</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonRightSide">
            <property name="text">
             <string>Okno prawe</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupSide</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonWindows">
          <property name="text">
           <string>Windows</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupAppType</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupAppType"/>
  <buttongroup name="buttonGroupSide"/>
 </buttongroups>
</ui>
